src/components/inputs/rich-text-input/utils/rules.js

```diff
@@ -0,0 +1,115 @@
+/* eslint-disable consistent-return */
```

Comment:
What does this module do? What's its job?

Reply:
They serialize and deserialize.

So if you have slate data, you can serialize it to HTML. And if you have HTML, you would deserialize it to slate format.